src/com/elikill58/negativity/spigot/SpigotNegativity.java

```diff
@@ -443,7 +443,7 @@ public static void sendAlertMessage(SpigotNegativityPlayer np, PlayerCheatAlertE
 						+ c.getName() + ") " + (alert.getNbAlertConsole() > 1 ? alert.getNbAlertConsole() + " times " : "") + "Reliability: " + reliability);
 		}
 		if (ProxyCompanionManager.isIntegrationEnabled()) {
-			sendAlertMessage(p, c.getName(), reliability, ping, alert.getHover().compile(c), alert.getNbAlert());
+			sendAlertMessage(p, c.getName(), reliability, ping, alert.getHover() == null ? null : alert.getHover().compile(), alert.getNbAlert());
```

Comment:
```suggestion
			sendAlertMessage(p, c.getName(), reliability, ping, alert.getHover() == null ? "" : alert.getHover().compile(), alert.getNbAlert());
```
Je ne suis pas sûr qu'AlertMessage supporte `null`.